### ISSUE TITLE
Reusing existing markup after server rendering

### ISSUE THREAD

**Issue by prontiol** [malte-wessel/react-custom-scrollbars#10]
React complains on the difference between server rendered markup, which is, basically:
```
<div style="position:relative;overflow:hidden;width:100%;height:100%;">
  <div style="position:relative;width:100%;height:100%;overflow:scroll;-webkit-overflow-scrolling:touch;">
  </div
</div>
```
and client-rendered one, which is the following:
```
<div style="position:relative;overflow:hidden;width:100%;height:100%;">
  <div style="position:absolute;top:0;left:0;right:-15px;bottom:-15px;overflow:scroll;-webkit-overflow-scrolling:touch;">
  </div>
</div>
```

Here is the error message:
```
Warning: React attempted to reuse markup in a container but the checksum was invalid. This generally means that you are using server rendering and the markup generated on the server was not what the client was expecting. React injected new markup to compensate which works but you have lost many of the benefits of server rendering. Instead, figure out why the markup being generated is different on the client or server:
 (client) div style="position:absolute;top:0;left:
 (server) div style="position:relative;width:100%;
```

**Comment by malte-wessel** [comment]
Thx again for reporting! This issue is due to the fact that we need to know the width and height of the browser's native scrollbars in order to hide them. On the server side we there are no scrollbars to check, so we take a width and height of 0. That leads to the different mark up on server and client. I'll need to investigate more on this.

**Comment by malte-wessel** [comment]
This is fixed in v3.0.0. Check out the [documentation on universal rendering](https://github.com/malte-wessel/react-custom-scrollbars/blob/master/docs/usage.md#universal-rendering) as well as the [upgrade guide](https://github.com/malte-wessel/react-custom-scrollbars/blob/master/docs/upgrade-guide-v2-v3.md)
